opensafely/jupyter.py

```diff
@@ -15,12 +16,16 @@
 
 # poor mans debugging because debugging threads on windows is hard
 if os.environ.get("DEBUG", False):
+
     def debug(msg):
         # threaded output for some reason needs the carriage return or else
         # it doesn't reset the cursor.
         sys.stderr.write("DEBUG: " + msg.replace("\n", "\r\n") + "\r\n")
         sys.stderr.flush()
+
+
```

Comment:
nit: I probably wouldn't have this whitespace here, but that's a personal style choice. If you prefer it, that's also fine.

Reply:
Huh, yes, odd choice on my part

Reply:
It was black!

Reply:
I'm innocent! :)